lib/msal-browser/.eslintrc.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
can we have an eslint package for all of our options so we don't have to copy them?

/lib/eslint-config-msal  

cc @jasonnutter

Reply:
Yes, I'm working on this for Angular.